releaseNotes/PythonFeatureManagement.md

```diff
@@ -2,6 +2,12 @@
 
 [Source code][source_code] | [Samples][samples]
 
+## 2.0.0b3 - November 14, 2024
+
+### Bug Fixes
+
+* Fixed an issue where targeted users that are exactly equal to 100 in allocation had the assignment reason of `None` instead of `Allocation` in telemetry.
```

Comment:
Fixed a bug where users exactly at the 100th percentile in allocation were incorrectly assigned a reason of `None` instead of `Allocation` in feature evaluation telemetry.